### ISSUE TITLE
Is the batch_hard Triplet Loss compatible with Keras?

### ISSUE THREAD

**Issue by chaitjo** [VisualComputingInstitute/triplet-reid#25]
With reference to the code [here](https://github.com/VisualComputingInstitute/triplet-reid/blob/master/loss.py), can I use this loss function as plug-and-play, or is there more work required to make this work for different model architectures and tasks?

**Comment by lucasb-eyer** [comment]
Hi, since Keras is just TensorFlow (if you're using the TensorFlow backend and "functional Keras"), yes, this code is compatible.

If you have more specific questions, please be more specific and reopen this issue.
